services/scheduler/src/config/manifest.json

```diff
@@ -4,9 +4,29 @@
     "name": "security",
```

Comment:
Add manifest.json validator to container.ts just like we validate the AppConfig (or initialUsersData from security service)